packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -168,6 +173,92 @@ function useProcessMedia({
     ]
   );
 
+  const trimExistingVideo = useCallback(
+    ({ resource: oldResource, start, end }) => {
+      const { src: url, mimeType } = oldResource;
+
+      const trimData = {
+        original: oldResource.id,
+        start,
+        end,
+      };
+
+      const onUploadStart = () => {
+        updateExistingElements({
+          oldResource: {
+            ...oldResource,
+            trimData,
+            isTrimming: true,
+          },
+        });
+      };
+
+      const onUploadError = () => {
+        updateExistingElements({
+          oldResource: { ...oldResource, isTrimming: false },
+        });
+      };
+
+      const onUploadSuccess = ({ resource }) => {
+        copyResourceData({ oldResource, resource });
+        updateOldMutedObject(oldResource.id, resource.id);
```

Comment:
This function seems wrongly named now, that it's also used for trimmed videos. Maybe rename to `updateOldTranscodedObject`?